tracks/python/exercises/isbn-verifier/mentoring.md

```diff
@@ -41,8 +41,8 @@ for position, value in enumerate(values)):
     total += int(value) * (10 - position)
 ```
 
-Generator expressions are considered very Pythonic and enable compact and clean iteration.
-They are also very efficient.
+Generator expressions are considered very Pythonic.
+Generators were added to the language (see [PEP-255]) becuase they provide a compact, clean and efficient method for iterating through data (compared to other approaches such as building lists or iterators).
```

Comment:
```suggestion
Generators were added to the language (see [PEP-255]) because they provide a compact, clean and efficient method for iterating through data (compared to other approaches such as building lists or iterators).
```